packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -41,13 +41,15 @@ const isArray = Array.isArray;
 let didWarnAboutStateAssignmentForComponent;
 let didWarnAboutUndefinedDerivedState;
 let didWarnAboutUninitializedState;
+let didWarnAboutGetSnapshotBeforeUpdateWithoutDidUpdate;
 let didWarnAboutLegacyLifecyclesAndDerivedState;
 let warnOnInvalidCallback;
 
 if (__DEV__) {
   didWarnAboutStateAssignmentForComponent = {};
   didWarnAboutUndefinedDerivedState = {};
   didWarnAboutUninitializedState = {};
+  didWarnAboutGetSnapshotBeforeUpdateWithoutDidUpdate = new Set();
```

Comment:
Personally I find it a bit confusing that one of them has a different type. Maybe convert them to sets (or even create a helper for this pattern) later?

Reply:
Sure! I updated the old objects to be sets (ca09ef8)